ibm/mas_devops/roles/sls_install/tasks/main.yml

```diff
@@ -161,12 +162,24 @@
 # -----------------------------------------------------------------------------
 - include_tasks: "tasks/sls-verify.yml"
 
+# Adding some delay to get SLS LicenseService ready
+# -----------------------------------------------------------------------------
+- name: Pause for 5 minute
+  pause:
+    minutes: 5
 
 # 9. Generate SLSCfg for MAS
 # -----------------------------------------------------------------------------
 - include_tasks: tasks/slscfg.yml
   when:
     - mas_instance_id is defined
-    - mas_instance_id != ""
+    - mas_instance_id|length > 0
     - mas_config_dir is defined
-    - mas_config_dir != ""
+    - mas_config_dir|length > 0
+
+# 10. Create Cloud CredentialsRequest in SLS namespace
```

Comment:
What is this for, what does it do?  Provide comments in this task and update the role README with details.

Reply:
We are validating the length of mas_config_dir over here.